backend/molgenis-emx2-sql/src/test/java/org/molgenis/emx2/sql/SqlQueryTest.java

```diff
@@ -0,0 +1,75 @@
+package org.molgenis.emx2.sql;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.withSettings;
+import static org.molgenis.emx2.Order.ASC;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import org.jooq.Condition;
+import org.jooq.DSLContext;
+import org.jooq.OrderField;
+import org.jooq.Record;
+import org.jooq.Result;
+import org.jooq.SelectConditionStep;
+import org.jooq.SelectFieldOrAsterisk;
+import org.jooq.SelectJoinStep;
+import org.jooq.SelectSeekStep1;
+import org.jooq.SelectSelectStep;
+import org.jooq.Table;
+import org.jooq.TableLike;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.molgenis.emx2.Column;
+import org.molgenis.emx2.Row;
+
+class SqlQueryTest {
+  @Test
+  void orderByTest() {
+
+    SqlSchemaMetadata schemaMock = Mockito.mock(SqlSchemaMetadata.class);
```

Comment:
Why don't you use a real schemametadata instance? There is no use to mock this?

This takes much more code and is much harder to follow what you are actually testing.